### PR TITLE
[IDE] Remove linked files from project

### PR DESCRIPTION
**Pull request by ketttY**
- For linked files only provide "Remove" (bound with cmd + delete shortcut) like VS Win
Screen Shot 2019-12-10 at 2 23 57 PM
VS Win:
Unknown

Fixes: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1026629/

### CODE REVIEW

**Comment by mrward** [comment]
Still seems to be some discussion from the design team about the behaviour here - I guess after this code change was made.

**Comment by ketttY** [comment]
@mrward Oops... Just saw that :) Thanks for the heads-up, adding do-not-merge label for now

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads.ProjectPad/ProjectFileNodeBuilder.cs [hunk 1]
```diff
@@ -285,32 +285,51 @@ public override void DeleteMultipleItems ()
...
 		}
 
+		[CommandUpdateHandler (EditCommands.Delete)]
+		[AllowMultiSelection]
+		void OnUpdateDeleteMultipleItems (CommandInfo info)
+		{
+			foreach (var node in CurrentNodes) {
+				var pf = (ProjectFile)node.DataItem;
+				if (pf.IsLink)
+					info.Text = "Remove";
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads.ProjectPad/ProjectFileNodeBuilder.cs]: This text needs to be localized

**Comment by ketttY** [comment]
@monojenkins rebase

**Pull request merged**